finesse/FINESSE_USAGE.md

```diff
@@ -100,3 +101,49 @@ Accuracy statistical summary:
 
 This example shows how the CLI Output of the tool, analyzing search results from
 Azure Search and providing an accuracy score for Finesse.
+
+## Scripts
+
+### XLSX Converter to JSON 📄
+
+This script converts data from an Excel file (.xlsx) into JSON format. It is
+used for questions generated created by non-developers. It is more readable an
+excel than a json file.
```

Comment:
Excel files are easier to read than JSON files.

Reply:
Done